src/models/TalkMapper.php

```diff
@@ -355,6 +382,137 @@ public function save($data) {
     }
 
     /**
+     * Edit a talk.
+     *
+     * Accepts a subset of talk fields
+     *
+     * The data-array is expected to have the following keys:
+     *
+     * * talk_title
+     * * url_fiendly_talk_title
+     * * talk_description
+     * * slides_link
+     * * language (a value from the column lang:lang_name
+     * * start_date (a timestamp)
+     * * duration
+     * * speakers (an array of names)
+     * * category (a value from the column categories:title)
```

Comment:
Isn't this `type`, not `category` ? (I'm looking at https://github.com/joindin/joindin-api/pull/186/files#diff-e27b217374c8436f4b830c74ed46cbbaR323)